server/views/partials/layout.njk

```diff
@@ -66,15 +65,24 @@
   {% endblock %}
 </title>
 
-
-
 {% block header %}
   {% include "./header.njk" %}
 {% endblock %}
 
 {% block bodyStart %}
 {% endblock %}
 
+{% block beforeContent %}
+  {% if not noBackLink %}
```

Comment:
minor one you might have already thought about, the double negative is a bit of a code smell since they're hard to read. May be fine if the alternatives are too much work, worth having a think about though 🙂

Reply:
Hmm, yea agree will sort this asap

Reply:
I think a positive default is one of the exceptions for the smell, not sure how difficult it is assume true within the template if it's not set and have the variable named something like 'displayBackLink' - it's okay to leave if it's a lot of hassle 😄